pages/manual/planet/overview.md

```diff
@@ -49,7 +49,7 @@ When you are inside PLANET you will see different features available.
 
 ![Planet Dashboard](images/planet-dashboard.png)
 
-List of features:
+<!--List of features:
```

Comment:
If you want to remove it then you should remove the code not comment it out.

Reply:
@lmmrssa Okay I'll do that!! Just wanted to keep it in there until someone approves of it.

Reply:
@lmmrssa I made the necessary changes. Please review it.